WordPressComKit/Me.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public class Me {
```

Comment:
Addressing immutability and/or structs here - https://github.com/Automattic/WordPressComKit/issues/2